src/styles/blocks/_header.scss

```diff
@@ -27,8 +27,12 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: color $transitionduration;
   }
 
+  &__item:hover {
+    color: #00acdc;
+  }
   &__item:not(:last-child) {
     margin-right: 20px;
   }
```

Comment:
instead of margin, you can use gap for grid displays

Reply:
yeah, @Alireza-Mohammadhossein is right here, gap will do better here